packages/router/test/router_preloader.spec.ts

```diff
@@ -196,6 +200,272 @@ describe('RouterPreloader', () => {
            })));
   });
 
+  describe('should support preloading stratages', () => {
+    let delayLoadUnPasued: BehaviorSubject<string[]>;
+    let delayLoadObserver$: Observable<string[]>;
+    let events: Array<RouteConfigLoadStart|RouteConfigLoadEnd>;
+
+    const subLoadChildrenSpy = jasmine.createSpy('submodule');
+    const lazyLoadChildrenSpy = jasmine.createSpy('lazymodule');
+    const buildPreloader = (
+        delayObserver$: Observable<string[]>,
+        ): typeof PreloadingStrategy => {
```

Comment:
```suggestion
        ): PreloadingStrategy => {
```

I think this might be easier to understand if you use `useFactory` instead and have the builder just return an instance.